rustls/src/crypto/mod.rs

```diff
@@ -24,15 +24,15 @@ pub use crate::rand::GetRandomFailed;
 pub use crate::msgs::handshake::KeyExchangeAlgorithm;
 
 /// Pluggable crypto galore.
-pub trait CryptoProvider: Send + Sync + 'static {
+pub trait CryptoProvider: Send + Sync + Debug + 'static {
```

Comment:
Unfortunately adding the `Debug` constraint in this commit without updating the provider example at the same time means this commit won't build+pass tests in isolation.

Reply:
squashed this and the next commit